packages/styles/scss/components/masthead/_masthead-l1.scss

```diff
@@ -12,6 +12,28 @@
 $search-transition: cubic-bezier(0.2, 0, 0.38, 0.9);
 $search-transition-timing: 95ms;
 
+@mixin short-masthead-l1 {
+  height: carbon--mini-units(6);
+}
+
+@mixin short-masthead-l1__name {
+  min-width: rem(208px);
+  padding-left: 0;
+}
+
+@mixin short-masthead-l1__eyebrow {
+  display: none;
+}
+
+@mixin short-masthead-l1__title {
+  line-height: inherit;
+  padding-bottom: $spacing-05;
+}
+
+@mixin short-masthead-l1__title-after {
+  bottom: 0;
+}
+
```

Comment:
I still can't think of anything other than mixins for sharing the same css without duplicating. I'd appreciate if any one suggests a different approach.

Reply:
Not sure if it would be cleaner to just have a single parent modifier class that targets each of these elements? seems a bit overengineered to have several mixins for this...

Reply:
@jeffchew Are you suggesting contextual selector here? Like defining css rules for elements under `.bx--masthead__l1--short`?

Reply:
yes, that's what I was thinking. Adding other folks here to chime in.
cc: @kennylam @annawen1 @photodow @sannminwin @andysherman2121